src/org/jitsi/meet/test/FollowMeTest.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright @ 2016 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test;
+
+import junit.framework.*;
+import org.jitsi.meet.test.util.*;
+
+import org.openqa.selenium.*;
+
+/**
+ * A test which tests "Follow Me" feature that allows moderators to enable 
+ * a mode in the conference where all participants are seeing what the 
+ * moderator is seeing.
+ *
+ * @author Kostiantyn Tsaregradskyi
+ */
+public class FollowMeTest
+    extends TestCase
+{
+    private final static String filmStripXPath = "//div[@id='remoteVideos']";
+    private final static String etherpadXPath = "//div[@id='etherpad']/iframe";
+    private final static String followMeCheckboxXPath =
+            "//input[@id='followMeCheckBox']";
+
+    /**
+     * Constructs test.
+     * @param name the method name for the test.
+     */
+    public FollowMeTest(String name)
+    {
+        super(name);
+    }
+
+    /**
+     * Orders the tests.
+     * @return the suite with order tests.
+     */
+    public static junit.framework.Test suite()
+    {
+        TestSuite suite = new TestSuite();
+
+        suite.addTest(new FollowMeTest(
+                "testFollowMeCheckboxVisibleOnlyForModerator"));
+        suite.addTest(new FollowMeTest(
+                "testShareDocumentCommandsAreFollowed"));
+        suite.addTest(new FollowMeTest(
+                "testFilmstripCommandsAreFollowed"));
+        suite.addTest(new FollowMeTest(
+                "testNextOnStageCommandsAreFollowed"));
+
+        return suite;
+    }
+
+    /**
+     * Checks that "Follow me" checkbox is only visible for moderator.
+     */
+    public void testFollowMeCheckboxVisibleOnlyForModerator()
+    {
+        System.err.println("Start testFollowMeCheckboxVisibleOnlyForModerator");
+
+        WebDriver owner = ConferenceFixture.getOwner();
+        WebDriver secondParticipant = ConferenceFixture.getSecondParticipant();
+
+        checkFollowMeCheckboxVisible(owner, true);
+        checkFollowMeCheckboxVisible(secondParticipant, false);
+
+        owner.findElement(By.id("followMeCheckBox")).click();
+    }
+
+    /**
+     * Checks if launching and then exiting Etherpad is executed for the second
+     * participant.
+     */
+    public void testShareDocumentCommandsAreFollowed() {
```

Comment:
This test will fail if no etherpad is configured, but we still want follow me to be tested. You can check: https://github.com/jitsi/jitsi-meet-torture/blob/master/src/org/jitsi/meet/test/EtherpadTest.java#L84
Currently pr-testing has no etherpad configured and pushing this without a check that skips the test will make all pr-testing to fail. Could you please add this check?